enclave-tls/samples/enclave-tls-client/echo.go

```diff
@@ -1,8 +1,10 @@
 package main
 
+// cgo LDFLAGS NOT support --enable-new-dtags which can use runpath instead of rpath
```

Comment:
Change to `The LDFLAGS defined by cgo doesn't support runtpath enabled by --enable-new-dtags, so rpath is used by default`